README.md

```diff
@@ -55,6 +55,149 @@ Edit the file `cdc-accelerator.json` in the project directory and add the `sourc
+```
+
+### How to use filters on preview
+
+The filter argument allows the user to filter the screens he wants to see using the apiKeys that are configured on the configuration file cdc-accelerator.json, for example:
+
+```
+ [{
+    apiKey: '1_2ABCDEFGHI345',
+    screens: [{ screenSetID: 'PreferencesCenter-ProfileUpdate', screenID: 'gigya-update-profile-screen' }],
+    emails: [ { emailID: 'codeVerification', languages: ['en'] } ]
+  }]
+```
+
+Here we are dealing with the apiKey '1_2ABCDEFGHI345', the filter is saying that the screen the user will see is the 'PreferencesCenter-ProfileUpdate' and the email template will be 'codeVerification' on that apiKey
+If the user wants to use the filter on more than one apiKey he can use the filter like this:
+
+```
+[{
+    apiKey: '1_2ABCDEFGHI345',
+    screens: [
+      { screenSetID: 'PreferencesCenter-ProfileUpdate', screenID: 'gigya-update-profile-screen' },
+      { screenSetID: 'PreferencesCenter-Landing', screenID: 'gigya-login-screen' },
+    ],
+    emails: []
+  },
+  {
+    apiKey: '1_3AS9DJAKSLA12',
+    emails: [{ emailID: 'doubleOptIn', languages: ['ar', 'en', 'pt-br'] } ]
+  }]
+```
+
+Here the first apiKey will have the webScreenSets filtered and on the second apiKey it will only show the emailTemplate 'doubleOptIn'.
+If the user needs to add that filter to all the ApiKeys that are on the configuration file, he can simply replace the apikey ID to '\*' for example:
+
+```
+[{
+  apiKey: '*',
+  emails: [{ emailID: 'doubleOptIn', languages: ['ar', 'en', 'pt-br'] } ]
+}]
+```
+
+By using this, the screen filtering will be applied to all the apiKeys inside the configuration file then the filter will then be applied on the preview file.
+
+### Using different options of preview
+
+Using the different options of the preview will enable the user to control what he wants to see or filter.
+
+The option <origin> will retrieve the settings that are available on the 'source' or 'deploy' inside the cdc-accelerator.json.
+The option <useLocalWebSdk> will use the local webSdk.js code that is inside the build/ directory.
+The option <useLocalScreenSets> will use the local screensets.js code that is inside the build/ directory
+The option <filter> will be what was defined above, with the specific apiKeys and screens/email.
+The option <lang> will define the language of the screen-sets, it can be changed accordingly to the user preference.
+
+```
+  preview({
+                origin: 'source',
+                useLocalWebSdk: true,
+                useLocalScreenSets: true,
+                filter,
+                lang: 'en',
+            })
+```
+
+### How to use specific environment commands and other functionalities of the CLI
+
+The user can use the CLI (Command Line Interface) to execute all operations instead of using the npm scripts.
+To use the CLI commands simply open the terminal and type npx cdc <operation>.
+The operation can be replaced by Init, Reset, Build, Deploy.
```

Comment:
The name of the operations should be lower case, Init, Reset, Build, Deploy are not accepted